src/cocotb/logging.py

```diff
@@ -75,9 +72,17 @@ def default_config() -> None:
     logging.basicConfig()
     logging.getLogger().handlers = [hdlr]  # overwrite default handlers
 
+    logging.getLogger("cocotb").setLevel(logging.INFO)
+    logging.getLogger("gpi").setLevel(logging.INFO)
+
 
-def _init(_: object) -> None:
-    """Set cocotb and pygpi log levels."""
+def _init() -> None:
+    """cocotb-specific logging setup.
+
+    Initializes the GPI logger and sets up the GPI logging optimization.
```

Comment:
"GPI logging optimization" probably needs context.

Reply:
There's some in the comments a couple lines below?